devtools/fixture_collector.py

```diff
@@ -79,14 +110,150 @@ async def main() -> None:
                 )
             )
 
-    print(f"Fixtures written to {target_dir}")
+    if not clean or verbose:
+        print(f"Fixtures written to {target_dir}")
 
     zip_file_name = f"{target_dir}.zip"
     with zipfile.ZipFile(zip_file_name, "w") as zip_file:
         for file_name in os.listdir(target_dir):
             zip_file.write(os.path.join(target_dir, file_name), file_name)
+            if clean:
+                os.remove(os.path.join(target_dir, file_name))
```

Comment:
All this I/O should be running in an executor thread, but since we don't care if we block the fixture collector for the duration of the write since nothing else is running it doesn't really matter